__tests__/tests/components/ImageZoom.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import { ImageZoom } from 'components';
+
+describe('<ImageZoom />', () => {
+  let wrapper;
+
+  const style = { backgroundColor: 'red' };
+  const uri = { uri: 'dummy.png' };
+
+  beforeEach(() => {
+    wrapper = shallow(<ImageZoom uri={uri} style={style} />);
+  });
```

Comment:
Using a lot of `beforeEach`'s can get a little confusing (usually it's used for environment configuration, such as initiating a database or something). I think a new wrapper should be defined inside of each test.